### DISCUSSION TITLE
How to prevent Githubs data hungry behavior

### DISCUSSION THREAD

**Discussion by zarat** [community/community#66462] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there no way to dismiss 2FA? I would really love to stay on Github, but if you require me to give you more personal information as needed i will go to Huggingface.

**Comment by ghostinhershell** [comment]
Hi @zarat 👋🏾 

Thanks so much for reaching out with your feedback. At the end of 2022, it was [announced on our blog](https://github.blog/2022-12-14-raising-the-bar-for-software-security-next-steps-for-github-com-2fa/) that GitHub will require all users who contribute code on GitHub.com to enable one or more forms of two-factor authentication (2FA) by the end of 2023. We started the phased roll-out in March 2023 -in order to learn about the efficacy and adjust as we scaled to larger groups. 

As more users are required to use it, we’ve noticed questions, comments, and feedback in the community. We’re here to help make adopting 2FA on your GitHub account as straightforward as possible.  For these and additional information, see the FAQ in our docs:
- [About two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication)
- [Configuring two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)

**Comment by zarat** [comment]
I already know (and mentioned in my question) that you force me to do so. My question was, can i avoid it in any way? I dont want it, i want to make decisions on my own. I am not a child.
